### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by ZabihullahNooriWardak**
**In this branch I have done the following tasks:**

1. Readme file created.
2. Linter configured for ES and CSS.
3. React components created for UI.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import AddBook from './components/addBookForm';
+
+function App() {
+  return (
+    <>
+      <Header />
+      <BookList />
+      <AddBook />
+    </>
+  );
```
Comment (iambenkis) [inline src/App.js:12]: - [ ] Good job create components for your bookstore 👏. In your App component, you should have routes , to ensure navigation between `Book` and `categories`

Capture

```jsx
// ./src/App.js

import React from "react"
import { BrowserRouter, Routes, Route, Link } from "react-router-dom"

function App() {
  return (
    <BrowserRouter>
      <Navbar />
      <Routes>
        <Route path="/" element={<Books />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
    </BrowserRouter>
  )
}
```


- [ ] Your app should contain 2 page components: `<Books>` and `<Categories>`, Kindly add them 👍

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @ZabihullahNooriWardak 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  <BookList> created
✔️ <NewBookForm> created 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Header from './components/Header';
+import BookList from './components/bookList';
+import Addbook from './components/addBookForm';
+
+function App() {
+  return (
+    <>
+      <Header />
+      <BookList />
+      <Addbook />
```
Comment (tugsem) [inline src/App.js:10]: - [ ] As displayed by the previous reviewer, please wrap your routable components into router. This has to be in the app component as it's the parent of your app and watches the route changes to display the different pages on a single page.
- [ ] There should be categories route after the booklist route, the form should be rendered under the booklist component. Kindly implement that and ensure your app is updating upon route changes.

**Review by tugsem — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️
Hello @ZabihullahNooriWardak 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Highlights 🚀
✔️Good job on working on the requested changes

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<BookList />} />
+          <Route path="/header" element={<Header />} />
+          <Route path="/addbook" element={<AddBook />} />
+        </Routes>
+      </BrowserRouter>
+    </>
+  );
+}
```
Comment (jerryowusu) [inline src/App.js:23]: As stated by the previous reviewer, kindly ensure that your components are wrapped with the react router in a correct way and they are working fine. Please take your time and work on the following which were requested by the previous reviewer: 

- [ ] The app component should have two routes, the categories and the booklist route. (kindly remove the Header and AddBook routes on lines 17 and 18)
- [ ] The form (AddBook) component should be rendered in the booklist component and not here
- [ ] The routes should work such that upon clicking on the Book or Categories in the header, it changes and display the different pages.

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @ZabihullahNooriWardak , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

soclose2


 ## Highlights
Great effort in making the requested changes :+1: 
All linters passed :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] It would be nice to provide a more descriptive name for your branch instead of `feature`. This will help make your work more professional and help know exactly what each branch is about. :pray: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @ZabihullahNooriWardak , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done implementing the requested changes. Your project is now working as expected ✔
- The milestone requirements have been met with success. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**